octoprint_mrbeam/static/js/design_store.js

```diff
@@ -279,8 +296,8 @@ $(function () {
             setTimeout(function () {
                 refreshButtonElement.text(refreshButtonText);
             }, 3000);
-            document.getElementById("design_store_iframe").src =
-                self.DESIGN_STORE_IFRAME_SRC;
+            document.getElementById("design_store_iframe").src = "#";
```

Comment:
why are we setting the src to # all the time on reload? 😕 is this working?

Reply:
Yes because the page is set not to be reloaded every time we click on the "design store" item in the menu based on the element source. So if we didn't set it to "#" it will not initialise

Reply:
@Yelko-mrbeam